src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -207,10 +237,9 @@ public static RootCause LocalizeRootCause(this AnomalyDetectionCatalog catalog,
             host.CheckUserArg(beta >= 0 && beta <= 1, nameof(beta), "Must be in [0,1]");
             host.CheckUserArg(rootCauseThreshold >= 0 && rootCauseThreshold <= 1, nameof(beta), "Must be in [0,1]");
 
```

Comment:
The name of the parameter should be rootCauseThreshold, could you help fix it in this PR? Thank you so much